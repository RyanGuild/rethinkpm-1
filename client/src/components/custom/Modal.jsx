import React from 'react';
import PropTypes from 'prop-types';

const Modal = (props) => {
  return (
    <div className="popup-modal">
      <div className="modal-content">
        <span
          id="closeBtn"
          className="closeX closeBtn "
          onClick={props.closeMod}
        >
          &times;
        </span>
        <form className="modal-form">
          <div className="form-group col-md-6">
            <h4>Set Appointment Date & Time</h4>
          </div>
          <div className="form-row">
            <div className="form-group col-md-6">
              <label htmlFor="prospectName">Prospect Name</label>
              <input type="text" className="form-control" />
            </div>
            <div className="form-group col-md-6">
              <label htmlFor="item2"> Item 2</label>
              <input type="text" className="form-control" />
            </div>
          </div>
        </form>
      </div>
    </div>
  );
};

Modal.propTypes = {};

export default Modal;
