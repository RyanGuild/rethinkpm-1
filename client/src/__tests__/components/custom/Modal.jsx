import React from 'react';
import render from '../../../__utils__/renderHelper';
import Modal from '../../../components/custom/Modal';

test('render and snapshot', () => {
  let {container} = render(() => <Modal />);
  expect(container).toMatchSnapshot();
});
