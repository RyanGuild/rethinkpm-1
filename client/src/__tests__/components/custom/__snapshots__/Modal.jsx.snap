// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render and snapshot 1`] = `
<div>
  <div
    class="popup-modal"
  >
    <div
      class="modal-content"
    >
      <span
        class="closeX closeBtn "
        id="closeBtn"
      >
        Ã—
      </span>
      <form
        class="modal-form"
      >
        <div
          class="form-group col-md-6"
        >
          <h4>
            Set Appointment Date & Time
          </h4>
        </div>
        <div
          class="form-row"
        >
          <div
            class="form-group col-md-6"
          >
            <label
              for="prospectName"
            >
              Prospect Name
            </label>
            <input
              class="form-control"
              type="text"
            />
          </div>
          <div
            class="form-group col-md-6"
          >
            <label
              for="item2"
            >
               Item 2
            </label>
            <input
              class="form-control"
              type="text"
            />
          </div>
        </div>
      </form>
    </div>
  </div>
</div>
`;
